File: dota-zone/dotazone/src/main/java/br/com/dotazone/model/service/AdapterAction.java
package br.com.dotazone.model.service;

import java.util.List;

import br.com.dotazone.model.entity.Hero;
import br.com.dotazone.model.entity.Item;

public interface AdapterAction {


    public void initList();

    public void initListItem(List<Item> items);

    public void initListIHero(List<Hero> heroes);

    public void initRating();
}

File: dota-zone/dotazone/src/main/java/br/com/dotazone/model/service/AdapterAction.kt
package br.com.dotazone.model.service

import android.view.View
import br.com.dotazone.model.entity.Hero;
import br.com.dotazone.model.entity.Item;


interface AdapterAction {
	fun initList(view: View?)
	fun initListItem(items: List<Item>)
	fun initListIHero(heroes: List<Hero>)
	fun initRating()
}
